docs/sphinx/_ext/dagster-sphinx/dagster_sphinx/docstring_flags.py

```diff
@@ -100,12 +106,9 @@ class flag(nodes.Element):
 
 def visit_flag(self, node: flag):
     flag_type, message = [node.attributes[k] for k in FLAG_ATTRS]
-    # We are currently not parsing the content of the message, so manually sub
-    # all `references` with `<cite>` tags, which is what the HTML writer does
-    # for parsed RST.
-    message = re.sub(r"`(\S+?)`", r"<cite>\1</cite>", message)
     # The "hidden" elements are not visible on screen, but are picked up by the search
     # crawler to provide better structure to search results.
+    message_html = "\n".join([f"<p>{line}</>" for line in message])
```

Comment:
IMO it would look better if we keep the first sentence ("This API may break in future versions...") on the same line with the deprecated/experimental tag, then just the additional text in further paragraphs.